prow/config/jobs_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var podRe = regexp.MustCompile(`^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`)
 
-const testThis = "@k8s-bot test this"
+const testThis = "/test"
```

Comment:
Likewise, this change makes it that we validate that `/test` will trigger all the `always_run: true` jobs (and only those jobs).

However we also need to still validate that `@k8s-bot test this` triggers all the `always_run: true` jobs (and none of the others).

We no longer have this coverage (feel free to test both of these facts)